src/jumplink.F90

```diff
@@ -206,7 +206,9 @@ subroutine makestab
   ! Set up expansion segments for type 'SUB', 'DRP', and 'DRS' nodes.
 
   newn = 0
-
+  do k=1,maxjl
```

Comment:
Would it make sense to assign this as `knt(1:maxjl) = 0`?

Reply:
Interesting - I hadn't seen that shorthand notation before, and in fact there are probably numerous places throughout the library where a similar change could be made.  I can look into that more as part of a future update to the library.